src/components/Scenarios/FormFieldData.jsx

```diff
@@ -22,7 +22,7 @@ export default function getFormFieldData(metadata) {
     }))
       .sort((a, b) => a.text.localeCompare(b.text)),
 
-    itemList: Object.keys(items).map(k => [items[k], k]).filter(x => x[0].cost >= itemCost).map(x => ({
+    itemList: Object.keys(items).map(k => [items[k], k]).filter(x => x[0].cost >= itemCost && !x[1].includes('recipe_')).map(x => ({
```

Comment:
maybe you could move the map after the filter to avoid the need to double map?

Reply:
I'm not sure if that's possible. I need to filter by property name and value. The items are stored in an object and not an array.

Reply:
you can still use items[k] to get the item object

Reply:
whoops you're totally right. I had a total tunnel vision.